typescript/src/redemption.ts

```diff
@@ -55,29 +56,36 @@ export interface RedemptionRequest {
 
 /**
  * Requests a redemption from the on-chain Bridge contract.
```

Comment:
SInce we call `tBTCToken.requestRedemption` instead of `bridge.requestRedemption` now, I'd change this sentence to something like "Requests a redemption of tBTC into BTC".